R/Transformations.R

```diff
@@ -307,13 +333,22 @@ TADA_CalculateTotalNP <- function(.data, sum_ref, daily_agg = c("max", "min", "m
     # "ActivityStartDateTime", #does not make sense to include for daily agg
     "ActivityRelativeDepthName",
     "TADA.MonitoringLocationIdentifier",
-    "MonitoringLocationName",
+    "TADA.MonitoringLocationName",
     "TADA.LongitudeMeasure",
     "TADA.LatitudeMeasure",
     "ActivityMediaSubdivisionName",
     "TADA.ActivityMediaName",
     "TADA.ComparableDataIdentifier",
-    # "TADA.ResultMeasure.MeasureUnitCode",
+    # "TADA.ResultMeasure.MeasureUnitCode", # this may not need to be included since we have TADA.ComparableDataIdentifier
+    # consider adding columns below... 
```

Comment:
@wokenny13 FYI - these column additions were meant to address this issue with metadata being carried over. Maybe it can be addressed elsewhere: https://github.com/USEPA/EPATADA/issues/588